packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -2959,6 +2959,23 @@ class EditableTextState extends State<EditableText> with AutomaticKeepAliveClien
     });
   }
 
+  @override
+  void handleKeyboardUndo(String direction) {
+    if (_editableKey.currentContext == null)
+        return;
+    if (direction == 'undo') {
```

Comment:
Nit: Would it be better to send a boolean from the engine?  It would save a little bit of space at least.

If not, maybe parse the string into an enum or compare it to constants.